src/presets/cloudflare/wrangler/environment.ts

```diff
@@ -6,6 +6,8 @@
  * Source: https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/config/environment.ts
  */
 
+import type { Json } from "miniflare";
```

Comment:
We should avoid direct dependency. Can you please inline it?

Reply:
No problem 👍